dot/parachain/candidate-validation/messages.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package candidatevalidation
+
+type ValidateFromChainState struct {
+}
+
+type ValidateFromExhaustive struct{}
+
+type PreCheck struct{}
```

Comment:
Implement these? This seems like something that we could do as part of this PR

Reply:
I've implemented structures for these.